stories/topics/ref-aws-additional-configs-secure-controller-load-balancer.adoc

```diff
@@ -34,11 +34,12 @@ Ensure you are in the correct region.
 Ensure you are in the correct region.
 . Select the internal load balancer for {ControllerName}.
 . Click the *Listeners* tab.
-. Select the Listener, and then click btn:[Edit].
+. Select the Listener, and then click btn:[Add listener].
 . Select *HTTPS* for the protocol, and ensure the port is `443`.
+. Select *Forward* for the default actions, and ensure the target group is `{ControllerName}` instance group.
 . Select *listener settings > Default SSL/TLS certificate* and ensure that *From ACM* is selected.
```

Comment:
Is is listener or Listener?

Reply:
Here is the screenshot 
![image](https://github.com/ansible/aap-docs/assets/60555139/0a1663c2-a95d-4dfa-aa2e-ab1abb879b52)

Reply:
I added a `Secure` word.  It is `Secure listener settings` now.